platform/MCU/NRF51/SPARK_Firmware_Driver/src/hw_config.c

```diff
@@ -146,6 +147,9 @@ void FLASH_Begin(uint32_t sFLASH_Address, uint32_t fileSize)
 {
     //OTA_FLASHED_Status_SysFlag = 0x0000;
     //Save_SystemFlags();
+#if PLATFORM_ID==103
```

Comment:
You had it right. I only saw the (i) and not the i=1. I need to be more cateful. Embarrassing! :-P